<ui version="4.0" >
 <class>YaMultiContactConfirmationTipLabel</class>
 <widget class="QWidget" name="YaMultiContactConfirmationTipLabel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="bottomMargin" >
      <number>5</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Delete these contacts?</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="YaFilteredContactListView" name="contactListView" />
   </item>
   <item>
    <widget class="AnimatedStackedWidget" name="actionButtonsStackedWidget" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>16</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="actionButtonsPage" >
      <layout class="QVBoxLayout" >
       <item>
        <widget class="YaLinkButton" name="deleteButton" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="deleteConfirmationPage" >
      <layout class="QVBoxLayout" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QDialogButtonBox" name="deleteConfirmationButtonBox" >
         <property name="focusPolicy" >
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="standardButtons" >
          <set>QDialogButtonBox::NoButton</set>
         </property>
         <property name="centerButtons" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimatedStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>animatedstackedwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>YaLinkButton</class>
   <extends>QPushButton</extends>
   <header>yalinkbutton.h</header>
  </customwidget>
  <customwidget>
   <class>YaFilteredContactListView</class>
   <extends>QListView</extends>
   <header>yafilteredcontactlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
